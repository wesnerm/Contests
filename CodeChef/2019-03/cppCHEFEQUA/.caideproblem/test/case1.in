3
1 2 3
3 6 14