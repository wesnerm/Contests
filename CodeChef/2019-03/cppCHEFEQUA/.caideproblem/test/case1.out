5 499122172 499122179
